test/unit/lib/plugins/aws/package/compile/events/stream.test.js

```diff
@@ -1479,6 +1479,11 @@ describe('test/unit/lib/plugins/aws/package/compile/events/stream.test.js', () =
       const { awsNaming, cfTemplate } = await runServerless({
         fixture: 'function',
         configExt: {
+          provider: {
+            kinesis: {
+              consumerNamingMode: 'serviceSpecific',
```

Comment:
Can we confirm that new naming has picked up in one of the underlying tests?

Reply:
Yes, I can confirm that. Without these lines one test is failing:

```
% npx mocha test/unit/lib/plugins/aws/package/compile/events/stream.test.js

....

  24 passing (3s)
  2 pending
  1 failing

  1) test/unit/lib/plugins/aws/package/compile/events/stream.test.js
       regular
         "before all" hook for "TODO: should support ARN String for `arn`":
     ServerlessError: Starting with v4.0.0 Kinesis streams naming scheme will change. Adapt to new schema by setting`provider.kinesis.consumerNamingMode` property to `serviceSpecific`.
  More Info: https://www.serverless.com/framework/docs/deprecations/#KINESIS_CONSUMER_NAME_CONTAINING_SERVICE
```

After adding them, all tests pass.

Reply:
@preshetin I suggested adding below a test, in which we confirm that, indeed new naming is used.

Currently, we've introduced a feature that doesn't have a single confirmation test, and that's not great

Reply:
@medikoo  I think I get what you mean. Sure, I'll do this and let you know.

Reply:
@medikoo test added in https://github.com/serverless/serverless/pull/9706/commits/fa7282a102c76a468a59552d96fa839fe9540d85 - should it be like that?

Reply:
@preshetin We should rather access the generated CF resource, and confirm that consumer name has the _service_ name in it.

Additionally _service_ name in case of `runServerless` is generated automatically, so instead of enforcing its value via `service` property, it'll be nicer to simply retrieve it from returned `serverless` instance (you can see how it's accessed here: https://github.com/serverless/serverless/blob/34d922d3d8d33f6e17e697610fdef7f13003a2f3/test/unit/lib/plugins/aws/remove/index.test.js#L218-L241)

Reply:
Thanks for the hints @medikoo. Here is another attempt at https://github.com/serverless/serverless/pull/9706/commits/6184922931ca7361991574c6a5247267b50230a9